Makefile

```
@@ -378,8 +395,19 @@ release-alias-tag: # Adds the tag to the last build tag.
 release-notes: $(RELEASE_NOTES) ## Generate release notes
 	$(RELEASE_NOTES) $(RELEASE_NOTES_ARGS)
 
+.PHONY: templates
+templates: templates/cluster-template.yaml \
+	   templates/cluster-template-without-lb.yaml \
+	   templates/cluster-template-external-cloud-provider.yaml
+
+templates/cluster-template.yaml: kustomize/v1alpha6/default FORCE
```

This fails for me with:

```
$ make templates
hack/tools/bin/kustomize build "kustomize/v1alpha6/default" > "templates/cluster-template.yaml"
bash: line 1: hack/tools/bin/kustomize: No such file or directory
make: *** [Makefile:404: templates/cluster-template.yaml] Error 127
make: *** Deleting file 'templates/cluster-template.yaml'
```

I would suggest we add the `$(KUSTOMIZE)` target as a dependency, so the binary is always available

```suggestion
templates/cluster-template.yaml: kustomize/v1alpha6/default FORCE $(KUSTOMIZE)
```

The same applies to targets `templates/cluster-template-%.yaml`, `$(E2E_TEMPLATES_DIR)/cluster-template.yaml` and `$(E2E_TEMPLATES_DIR)/cluster-template-%.yaml`

---

I've updated with this change.